pkgs/development/guile-modules/guile-gcrypt/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, guile, libgcrypt
+, autoreconfHook, pkgconfig, texinfo
+}:
+
+let
+  name = "${pname}-${version}";
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+in stdenv.mkDerivation {
+  inherit name;
+
+  src = fetchurl {
```

Comment:
Is there a benefit to using `fetchurl` over `fetchgit` here?

Reply:
Is there a benefit to not? I presumed fetchurl would be the preferred default, happy to change otherwise!

Reply:
I don't think we really have a standard, but I prefer using the most specific tool for the job, except in cases where there is a reason to `fetchurl`/`fetchTarball` (e.g. when a release tarball contains things that will not be present if you build it from source). Ultimately up to you (same for all other instances of this).

Reply:
fair enough, will push in a few hours

Reply:
if a url is available, it should be preferred. fetchgit requires getting all of the git history and is more expensive.